cfme/cloud/instance.py

```diff
@@ -276,14 +276,16 @@ def on_details(self, force=False):
             else:
                 return True
 
-    def find_quadicon(self, do_not_navigate=False, mark=False, refresh=True):
+    def find_quadicon(self, do_not_navigate=False, mark=False, refresh=True, any=False):
         """Find and return a quadicon belonging to a specific instance
 
         Returns: :py:class:`cfme.web_ui.Quadicon` instance
         Raises: InstanceNotFound
         """
         if not do_not_navigate:
-            if not self.provider_crud.load_all_provider_instances():
+            if any:
```

Comment:
so by "any" do we really mean "all"? and isn't "any" a keyword, as such we shouldn't use it?